cmd/system-probe/main.go

```diff
@@ -158,18 +158,25 @@ func runAgent(exit <-chan struct{}) {
 }
 
 func enableProfiling(cfg *config.AgentConfig) error {
-	// allow full url override for development use
-	s := ddconfig.DefaultSite
-	if cfg.ProfilingSite != "" {
-		s = cfg.ProfilingSite
-	}
+	var site string
+	v, _ := version.Agent()
+
+	// check if TRACE_AGENT_URL is set, in which case, forward the profiles to the trace agent
+	if traceAgentURL := os.Getenv("TRACE_AGENT_URL"); len(traceAgentURL) > 0 {
```

Comment:
Should this be prefixed with`DD_` like most of our other environment variables?

Reply:
Nevermind. After discussing over slack, this is the environment variable used with the existing public profiling libraries, and we should be consistent.

Reply:
@leeavital this is not true. There is no such variable in any of APM integrations. All of our variables are prefixed `DD_`. In case of the agent, there is `DD_AGENT_HOST` and `DD_AGENT_PORT`, none of which are guaranteed to be set and are most likely not set on most systems (since they default to `localhost:8126`). Perhaps you could use that logic here too. Then again, you are already making a big assumption: that the tracing clients (or profiler) is running on the same machine as the system-probe, is that fair?